core/services/postgres/transaction.go

```diff
@@ -0,0 +1,39 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+)
+
+func GormTransaction(ctx context.Context, db *gorm.DB, fc func(tx *gorm.DB) error, txOptss ...sql.TxOptions) (err error) {
```

Comment:
Go's SQL primitives are designed so you can always `defer rollback` and if the transaction was committed first it'll do nothing, so I'm not sure this helper function gets you all that much?

Reply:
You might be right... I was just moving this from the `utils` package, though, so it's probably out of scope for this PR to change up our approach.  I'll look into it and will add a new story if it seems like a good approach.